stdlib/multiprocessing/dummy/__init__.pyi

```diff
@@ -33,7 +33,6 @@ Process = DummyProcess
 class Namespace:
     def __init__(self, **kwds: Any) -> None: ...
     def __getattr__(self, __name: str) -> Any: ...
-    def __setattr__(self, __name: str, __value: Any) -> None: ...
```

Comment:
This is not redundant. It prevents getting type check errors from `namespace.foobar = "some value"`. The same goes for `stdlib/multiprocessing/managers.pyi`.

Reply:
Thanks!